kickstarts/base.ks.erb

```diff
@@ -117,6 +117,9 @@ echo "" >> /etc/ssh/sshd_config
 # Remove packages not needed
 dnf -C -y --noplugins remove linux-firmware doxygen
 
+# Ensure packages are up to date
+dnf -y update manageiq-*
```

Comment:
This won't work with productized builds, I don't think.

Reply:
Fixed